samples/sampler/custom-addon/DefaultSkinToolbarTab.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'; // eslint-disable-line
+import {useGlobals} from '@storybook/api';
+import ToolbarTab from './ToolbarTab';
+import PropTypes from 'prop-types';
```

Comment:
imports should be grouped as module imports, and below are the local imports.

```suggestion
import {useGlobals} from '@storybook/api';
import PropTypes from 'prop-types';

import ToolbarTab from './ToolbarTab';
```

React import is not necessary. 
Apply this suggestion on the other files as well